bigstream/align.py

```diff
@@ -210,12 +212,15 @@ def feature_point_ransac_affine_align(
     nspots=5000,
     match_threshold=0.7,
     max_spot_match_distance=None,
+    point_matches_threshold=50,
```

Comment:
Thanks for removing these bad hard coded thresholds - this is much better. I'll want to add docstring entries for them. Also - FWIW - most of the time if too few spots are detected, it's much better to increase the number of spots found (using fix_spot_detection_kwargs or mov_spot_detection_kwargs - specifically `threshold` and/or `threshold_rel`) than to decrease the number of spots needed. Transforms found based on too few spots will almost always be bad alignments.

Reply:
I will add something here but feel free to change it to a better description after I merge